libraries/adopt_openjdk_helpers.rb

```diff
@@ -2,23 +2,30 @@ module Java
   module Cookbook
     module AdoptOpenJdkHelpers
       def sub_dir(url)
-        uri = URI.parse(url)
```

Comment:
Does it make sense to keep `URI.parse` to make sure the path provided is a valid URI? That way if someone mistakenly provided something like `foo\bar/bam.tar.gz` it would fail fast.

Reply:
Yeah, I think that'd be a great thing to keep at least.